### PR TITLE
chore: some database changes

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: coordinator/src/metrics.rs [hunk 1]
```diff
@@ -158,11 +158,12 @@ fn position_metrics(cx: &Context, node: &Node) {
         );
         match position.direction {
             Direction::Long => {
-                margin_long += position.collateral;
+                // TODO: fix me: this was meant to be the traders margin
```
Comment (bonomat) [inline coordinator/src/metrics.rs:161]: I'll create a follow-up ticket for this.

File: coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE positions
+    ADD trader_margin  BIGINT NULL;
+
+UPDATE positions set trader_margin = CAST(quantity AS FLOAT)  / (CAST(average_entry_price AS FLOAT) * CAST(trader_leverage as FLOAT)) * 100000000;
+
+ALTER TABLE positions
+ALTER COLUMN trader_margin SET NOT NULL;
```
Comment (holzeis) [inline coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql:7]: nit: missing newline

File: coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE positions
+    ADD trader_margin  BIGINT NULL;
```
Comment (holzeis) [inline coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql:2]: nit: one space too much 🙈

File: coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+UPDATE positions set trader_margin = CAST(quantity AS FLOAT)  / (CAST(average_entry_price AS FLOAT) * CAST(trader_leverage as FLOAT)) * 100000000;
```
Comment (holzeis) [inline coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql:4]: 🤔 I trust that this are the right amount of zeros? 😅

Comment (holzeis) [reply coordinator/migrations/2023-10-11-104835_add_trader_margin_to_positions/up.sql:4]: nit: one space too much 😅

**Review by holzeis — APPROVED** [review]
LGTM 👍

**Pull request merged**